date/src/date.yml

```diff
@@ -0,0 +1,15 @@
+name: date
+version: "0.0.0"
+author: "Jonas Geschke <mail@yonny.de>"
+about: "Display the current date and time."
+args:
+  - utc:
+      help: print or set Coordinated Universal Time (UTC)
```

Comment:
Prefer the word `display` over `print`/`prints`, also start help messages with upper letter

Reply:
okay updated it

Reply:
Nice, now it's nice :3